### PR TITLE
[with #1549] [Test] Add save load test

### PR DESCRIPTION
**Pull request by zhoonit**
**[Pending commit: #1549]**

- [Test] Add save load test

```
Add save load test while adding options to model test

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

```

### CODE REVIEW

File: test/unittest/unittest_nntrainer_models.cpp [hunk 1]
```diff
@@ -590,18 +618,71 @@ TEST_P(nntrainerModelTest, model_test_validate) {
...
+  auto nn = nntrainer::NeuralNetwork();
+  EXPECT_NO_THROW(nn.loadFromConfig(getIniName()));
+  EXPECT_NO_THROW(nn.compile());
+  EXPECT_NO_THROW(nn.initialize());
+
+  auto saved_ini_name = getName() + "_saved.ini";
+  if (remove(saved_ini_name.c_str())) {
+    /// do nothing
+  }
+  nn.save(saved_ini_name, ml::train::ModelFormat::MODEL_FORMAT_INI);
```
Comment (kparichay) [inline test/unittest/unittest_nntrainer_models.cpp:636]: EXPECT_NO_THROW or return == 0?

**Review by kparichay — APPROVED** [review]
LGTM
conv and pool both remain to support INI save right?

**Comment by zhoonit** [comment]
> LGTM
> conv and pool both remain to support INI save right?

Yes, @lhs8928 is working on it, he will enable this test :)

File: test/unittest/unittest_nntrainer_models.cpp (hunk 1 above)
Comment (zhoonit) [reply test/unittest/unittest_nntrainer_models.cpp:636]: updated

**Review by lhs8928 — APPROVED** [review]
LGTM

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**